eip155/caip25.md

```diff
@@ -65,16 +68,36 @@ Similarly, wherever possible, session properties should align closely with infor
       }
     },
     "optionalScopes":{
-      "eip155:42161": {
+      "eip155:84532": {
         "methods": ["eth_sendTransaction", "eth_signTransaction", "get_balance", "personal_sign"],
         "notifications": ["accountsChanged", "chainChanged"]
     },
     "sessionProperties": {
       "expiry": "2022-12-24T17:07:31+00:00",
       "caip154": {
-          "supported":"true"
+        "supported":"true"
+      },
+      "flow-control": {
+        "loose": [], 
+        "strict": [], 
+        "exoticThirdThing": [] //caller is requesting a configuration-set which the wallet will drop as unrecognized      
+      }
+    },
+    "scopedProperties": {
+      "8453": {
```

Comment:
why not caip2?

Reply:
whoops that was a mistake on my part, you're right that it should match the examples in the CAIP-25 spec and use full CAIP-2s